.github/CODEOWNERS

```diff
@@ -1,15 +1,15 @@
-*                         @matrix-org/element-web-reviewers
-/.github/workflows/**     @matrix-org/element-web-team
-/package.json             @matrix-org/element-web-team
-/yarn.lock                @matrix-org/element-web-team
+*                         @element-hq/element-web-reviewers
+/.github/workflows/**     @element-hq/element-web-team
+/package.json             @element-hq/element-web-team
+/yarn.lock                @element-hq/element-web-team
 
-/src/SecurityManager.ts                           @matrix-org/element-crypto-web-reviewers
-/test/SecurityManager-test.ts                     @matrix-org/element-crypto-web-reviewers
-/src/async-components/views/dialogs/security/     @matrix-org/element-crypto-web-reviewers
-/src/components/views/dialogs/security/           @matrix-org/element-crypto-web-reviewers
-/test/components/views/dialogs/security/          @matrix-org/element-crypto-web-reviewers
-/src/stores/SetupEncryptionStore.ts               @matrix-org/element-crypto-web-reviewers
-/test/stores/SetupEncryptionStore-test.ts         @matrix-org/element-crypto-web-reviewers
+/src/SecurityManager.ts                           @element-hq/element-crypto-web-reviewers
```

Comment:
Looks like this team doesn't exist on element-hq.

Should that be created or use [element-crypto](https://github.com/orgs/matrix-org/teams/element-crypto) for now @richvdh

Reply:
It should do, I just created it: https://github.com/orgs/element-hq/teams/element-crypto-web-reviewers

Reply:
I guess it can't be a secret team - I've changed it to public but it might take a while for GH to realise and update the codeowners file check.

Reply:
Nope, obviously it was because I forgot to add the team's access to the repo 🤦